packages/v4/src/content/design-guidelines/components/data-list/data-list.md

```diff
@@ -53,27 +53,29 @@ Every data list can be extended with these functionalities:
 * **[Expandable data list](/components/data-list/design-guidelines#expandable-data-list)**
 
 ### Actionable data list
-The actionable data list provides checkboxes that enable users to select one or more rows and act on that selection using options in the [toolbar](/components/toolbar/design-guidelines).
+The actionable data list provides checkboxes or radio buttons that enable users to select one or more rows in the list. Users may then act on those selections using options in the [toolbar](/components/toolbar/design-guidelines).
 
-<img src="./img/actionable-list.png" alt="Actionable list" /> 
+<img src="./img/actionable-data-list.png" alt="Actionable list" /> 
 
 1. **[Bulk selection](/guidelines/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/guidelines/bulk-selection) for more information. 
-2. **Checkbox:** Enables a user to select a row.
-3. **Global actions:** Actions that can be applied to all selected items.
+2. **Checkbox:** Enables a user to select a row. Use when multiple rows can be selected at the same time.
+3. **Radio button:** Enables a user to select a row. Use when only one row can be selected at a time.
```

Comment:
Even though this seems perfectly valid, we didn't actually implement this feature for data lists.

Reply:
Removed it from data list then!